packages/inventory/src/components/detail/TopBar.js

```diff
@@ -37,7 +42,21 @@ const TopBar = ({
             <SplitItem isFilled>
                 {
                     loaded ? (
-                        <Title headingLevel="h1" size='2xl'>{ entity && entity.display_name }</Title>
+                        <Flex>
+                            <FlexItem>
+                                <Title headingLevel="h1" size='2xl'>{ entity && entity.display_name }</Title>
+                            </FlexItem>
+                            {
+                                showTags &&
+                                <FlexItem>
+                                    {loaded ?
```

Comment:
This condition does not make sense. `loaded` must always be `true` at this point.